src/sections/General/Navigation/utility/menu-items.js

```diff
@@ -80,9 +80,32 @@ const Data = {
       // }
     },
     {
-      name: "Products",
+      name: "Products/Projects",
       path: "/projects",
       subItems: [
+        // {
+        //   name: "Image Hub",
+        //   path: "/projects/image-hub",
+        //   sepLine: true
+        // },
+        {
+          name: "PRODUCTS",
+          path: "/products",
```

Comment:
@aakashdeepsil can you navigate this link to meshmap for now, until we have products page up. This link is broken